patient_portal/get_patient_info.php

```diff
@@ -103,104 +100,121 @@
+        COL_ID,
+        COL_PID,
+        COL_POR_PWD,
+        COL_POR_SALT,
+        COL_POR_PWD_STAT
+    )) . " FROM " . TBL_PAT_ACC_ON . " WHERE " . COL_POR_USER . "=?";
+    $auth = privQuery($sql, array(
+        $_POST['uname']
+    ));
+    if ($auth === false) {
                 $logit->portalLog('login attempt','',($_POST['uname'].':invalid username'),'','0');
                 session_destroy();
-                header('Location: '.$landingpage.'&w');
-                exit;
+                header('Location: ' . $landingpage . '&w&u');
+                exit();
             }
             if(empty($auth[COL_POR_SALT]))
             {
                 if(SHA1($plain_code)!=$auth[COL_POR_PWD])
                 {
                     $logit->portalLog('login attempt','',($_POST['uname'].':pass not salted'),'','0');
                     session_destroy();
-                    header('Location: '.$landingpage.'&w');
-                    exit;                        
+                    header('Location: ' . $landingpage . '&w&p');
+                    exit();
                 }
                 $new_salt=oemr_password_salt();
                 $new_hash=oemr_password_hash($plain_code,$new_salt);
-                $sqlUpdatePwd= " UPDATE " . TBL_PAT_ACC_ON
-                              ." SET " .COL_POR_PWD."=?, "
-                              . COL_POR_SALT . "=? "
-                              ." WHERE ".COL_ID."=?";
-                privStatement($sqlUpdatePwd,array($new_hash,$new_salt,$auth[COL_ID]));   
-            }
-            else {
-                if(oemr_password_hash($plain_code,$auth[COL_POR_SALT])!=$auth[COL_POR_PWD])
-                {
+    $sqlUpdatePwd = " UPDATE " . TBL_PAT_ACC_ON . " SET " . COL_POR_PWD . "=?, " . COL_POR_SALT . "=? " . " WHERE " . COL_ID . "=?";
+    privStatement($sqlUpdatePwd, array(
+        $new_hash,
+        $new_salt,
+        $auth[COL_ID]
+    ));
+} else {
+    $tmp = oemr_password_hash($plain_code, $auth[COL_POR_SALT]);
+    if ($tmp != $auth[COL_POR_PWD]) {
                     $logit->portalLog('login attempt','',($_POST['uname'].':invalid password'),'','0');
                     session_destroy();
-                    header('Location: '.$landingpage.'&w');
-                    exit;                        
+        header('Location: ' . $landingpage . '&w&p');
+        exit();
 
                 }
 
             }
             $_SESSION['portal_username']=$_POST['uname'];
     $sql = "SELECT * FROM `patient_data` WHERE `pid` = ?";
 
-    if ($userData = sqlQuery($sql, array($auth['pid']) )) { // if query gets executed
+if ($userData = sqlQuery($sql, array(
+    $auth['pid']
+))) { // if query gets executed
 
         if (empty($userData)) {
             $logit->portalLog('login attempt','',($_POST['uname'].':not active patient'),'','0');
             session_destroy();
                             header('Location: '.$landingpage.'&w');
-            exit;
+        exit();
```

Comment:
Either syntax works here, but for a normal exit, if you want to put parens after, you might as well do exit(0);